---
layout: default
title: "Projects that use OBI"
permalink: /projects/
---

Contact us to add your ontology-driven project if it makes use of OBI!

### Genomic Epidemiology Ontology (GenEpiO), The FoodOn food ontology, and the Genomic Epidemiology Entity Mart (GEEM)

[genepio.org](https://genepio.org), [foodon.org](https://foodon.org), [genepio.org/geem](https://genepio.org/geem)

The Genomic Epidemiology Ontology (GenEpiO) project aims to develop a more comprehensive controlled vocabulary for infectious disease surveillance and outbreak investigations. To support foodborn disease informatics, FoodOn, an offshoot of GenEpiO and ENVO, describes food samples by organism, ingredient, packaging, food processing history etc.. GEEM is a software tool to take standards composed within ontologies according to OBI value specifications, and turn them into JSON, HTML form, and other formats for use in software to support data validation and exchange.



### Immune Epitope Database and Analysis Project

[iedb.org](https://iedb.org)

The Immune Epitope Database (IEDB) is a freely available resource funded by NIAID. It catalogs experimental data on antibody and T cell epitopes studied in humans, non-human primates, and other animal species in the context of infectious disease, allergy, autoimmunity and transplantation. The IEDB also hosts tools to assist in the prediction and analysis of epitopes.



### Human Immunology Project Consortium (HIPC)

[hipc.org](https://www.immuneprofiling.org/hipc)
The Human Immunology Project Consortium (HIPC) program was established in 2010 by the NIAID Division of Allergy, Immunology, and Transplantation as part of the overall NIAID focus on human immunology. Through this program, well-characterized human cohorts are studied using a variety of modern analytic tools, including multiplex transcriptional, cytokine, and proteomic assays; multiparameter phenotyping of leukocyte subsets; assessment of leukocyte functional status; and multiple computational methods. The HIPC program provides centralized research resources and comprehensive, centralized databases that utilize formal data standards and includes the Immunology Database and Analysis Portal (ImmPort) (https://www.immport.org/), offering advanced information technology support in the archiving and exchange of scientific data as an extensive data warehouse containing experimental data and metadata.



### Chemical Effects in Biological Systems (CEBS)

[CEBS Search](https://manticore.niehs.nih.gov/cebssearch)

The CEBS database houses data of interest to environmental health scientists. CEBS is a public resource, and has received depositions of data from academic, industrial, and governmental laboratories. CEBS is designed to display data in the context of biology and study design, and permit data integration across studies for novel meta analysis. OBI is used by CEBS to model studies, and improve search and analysis.



### Evidence and Conclusion Ontology (ECO)

http://evidenceontology.org/
The Evidence and Conclusion Ontology (ECO) has been developed to describe types of evidence and assertion methods. ECO terms are used in the process of biocuration to capture the evidence that supports biological assertions (e.g. gene product X has function Y as supported by evidence Z). The capture of evidence information allows tracking of annotation provenance, establishment of quality control measures, and precise query of annotations. ECO is in use by many leading biological resources including the Gene Ontology, UniProt and several model organism databases. ECO is freely available for download from GitHub (https://github.com/evidenceontology/) or the project's website (http://evidenceontology.org/).
ECO and OBI have an ongoing collaboration to create terms in OBI that correspond to assays and protocols underlying terms in ECO. In addition, ECO uses logical axioms linking to OBI assay terms to precisely define ECO terms.



### Common Fund Data Ecosystem (CFDE) Coordination Center

https://www.nih-cfde.org/
The Common Fund Data Ecosystem (CFDE) (https://commonfund.nih.gov/dataecosystem) is being developed to connect datasets, tools, and resources from the various Data Coordination Centers (DCCs) associated with Common Fund programs (e.g. HMP, KidsFirst, LINCS, GTEx, 4DN Nucleome, Metabolomics, SPARC, HubMAP, and MoTrPAC) This program will not only make the data more accessible but will facilitate cross-DCC queries and data analysis. As part of the CFDE, the CFDE Data Coordination Center provides a cloud-based infrastructure and a common metadata model to indefinitely house the data and tools associated with the Common Fund programs. The Coordination Center also provides a data query portal. The CFDE Coordination Center is using OBI assay terms to standardize metadata about the type of experiment that generated various datatypes (e.g. 16S amplicon sequence, whole genome sequence).
