---
layout: default
title: "Projects that use OBI"
permalink: /projects/
---

Contact us to add your ontology-driven project if it makes use of OBI!

### Cell Line Ontology (CLO)

[clo-ontology.org](http://www.clo-ontology.org/)

The Cell Line Ontology (CLO) is a community-driven ontology that is developed to standardize and integrate cell line information and support computer-assisted reasoning. The CLO development has becoming a community effort with international collaborations. Our development consortium includes experts from all over the world: the USA, Europe, and Japan.


### Chemical Effects in Biological Systems (CEBS)

[CEBS Search](https://manticore.niehs.nih.gov/cebssearch)


The CEBS database houses data of interest to environmental health scientists. CEBS is a public resource, and has received depositions of data from academic, industrial, and governmental laboratories. CEBS is designed to display data in the context of biology and study design, and permit data integration across studies for novel meta analysis. OBI is used by CEBS to model studies, and improve search and analysis.


### Chemical Methods Ontology (CHMO)

[CHMO](https://github.com/rsc-ontologies/rsc-cmo)

CHMO, the chemical methods ontology, describes methods used to collect data in chemical experiments, such as mass spectrometry and electron microscopy prepare and separate material for further analysis, such as sample ionisation, chromatography, and electrophoresis synthesise materials, such as epitaxy and continuous vapour deposition It also describes the instruments used in these experiments, such as mass spectrometers and chromatography columns. It is intended to be complementary to the Ontology for Biomedical Investigations (OBI).


### Clinical Epidemiology Resources (ClinEpiDB)

[clinepidb.org](www.clinepidb.org)

ClinEpiDB is an open-access online resource of population-based epidemiological studies addressing global-health concerns. ClinEpiDB aims to enable investigators to maximize the utility and reach of their data and to make optimal use of data released by others. It is associated with VEuPathDB and relies heavily on the OBI-based VEuPathDB ontology. Primary funding is from the Bill & Melinda Gates Foundation.


### Common Fund Data Ecosystem (CFDE) Coordination Center

[nih-cfde.org](https://www.nih-cfde.org/)

The Common Fund Data Ecosystem (CFDE) (https://commonfund.nih.gov/dataecosystem) is being developed to connect datasets, tools, and resources from the various Data Coordination Centers (DCCs) associated with Common Fund programs (e.g. HMP, KidsFirst, LINCS, GTEx, 4DN Nucleome, Metabolomics, SPARC, HubMAP, and MoTrPAC) This program will not only make the data more accessible but will facilitate cross-DCC queries and data analysis. As part of the CFDE, the CFDE Data Coordination Center provides a cloud-based infrastructure and a common metadata model to indefinitely house the data and tools associated with the Common Fund programs. The Coordination Center also provides a data query portal. The CFDE Coordination Center is using OBI assay terms to standardize metadata about the type of experiment that generated various datatypes (e.g. 16S amplicon sequence, whole genome sequence).


### An Ontology for Drug Discovery Investigations (DDI)

[DDI](http://code.google.com/p/ddi-ontology/)

The goal of DDI project is to develop an ontology for the description of drug discovery investigations. The Pharmaceutical industry is an information based industry. The aim of DDI is to add value to the information generated in the drug pipeline by making the information easier to reuse, integrate, curate, retrieve, and reason with. DDI aims to follow to the Open Biomedical Ontologies Foundry (OBO) principles, to use relations laid down in the OBO Relation Ontology, and to be compliant with Ontology for biomedical investigations (OBI).


### eagle-i

[eagle-i.net](www.eagle-i.net)

eagle-i is a resource discovery tool built to share information about research resources, giving any scientist, anywhere, access to one of the largest collections of information about Core Facilities, iPS cell lines, and other valuable, unique or rare scientific resources in the world.


### Eukaryotic Pathogen, Vector and Host Informatics Resources (VEuPathDB)

[veupathdb.org](www.veupathdb.org)

The Eukaryotic Pathogen, Vector and Host Informatics Resource (VEuPathDB) is a Bioinformatics Resource Center (BRC) funded by the US National Institute of Allergy and Infectious Diseases (NIAID). VEuPathDB focuses on eukaryotic pathogens and invertebrate vectors of infectious diseases. In addition to support from NIAID, VEuPathDB also received funding from the Wellcome Trust (UK), Bill & Melinda Gates Foundation, and US Department of Agriculture to support informatics efforts focusing on kinetoplastida and fungal organisms with special emphasis on improving functional annotation for select genome sequences and families of genes. VEuPathDB provides access to diverse genomic and other large scale datasets related to eukaryotic pathogens and invertebrate vectors of disease. Component web sites are constructed using a common infrastructure and standard data analysis and loading procedures, including the use of the VEuPathDB ontology based on OBI with integration of other OBO ontologies such as PATO, OGMS, DO, etc. as needed for coverage. 


### Evidence and Conclusion Ontology (ECO)

[evidenceontology.org](http://evidenceontology.org/)

The Evidence and Conclusion Ontology (ECO) has been developed to describe types of evidence and assertion methods. ECO terms are used in the process of biocuration to capture the evidence that supports biological assertions (e.g. gene product X has function Y as supported by evidence Z). The capture of evidence information allows tracking of annotation provenance, establishment of quality control measures, and precise query of annotations. ECO is in use by many leading biological resources including the Gene Ontology, UniProt and several model organism databases. ECO is freely available for download from GitHub (https://github.com/evidenceontology/) or the project's website (http://evidenceontology.org/).
ECO and OBI have an ongoing collaboration to create terms in OBI that correspond to assays and protocols underlying terms in ECO. In addition, ECO uses logical axioms linking to OBI assay terms to precisely define ECO terms.


### Gemma

[gemma.chibi.ubc.ca](http://gemma.chibi.ubc.ca)

Gemma is a web site, database and a set of tools for the meta-analysis, re-use and sharing of genomics data, currently primarily targeted at the analysis of gene expression profiles. Gemma contains data from thousands of public studies, referencing thousands of published papers. Users can search, access and visualize coexpression and differential expression results.


### Genomic Epidemiology Ontology (GenEpiO), The FoodOn food ontology, and the Genomic Epidemiology Entity Mart (GEEM)

[genepio.org](https://genepio.org), [foodon.org](https://foodon.org), [genepio.org/geem](https://genepio.org/geem)

The Genomic Epidemiology Ontology (GenEpiO) project aims to develop a more comprehensive controlled vocabulary for infectious disease surveillance and outbreak investigations. To support foodborn disease informatics, FoodOn, an offshoot of GenEpiO and ENVO, describes food samples by organism, ingredient, packaging, food processing history etc.. GEEM is a software tool to take standards composed within ontologies according to OBI value specifications, and turn them into JSON, HTML form, and other formats for use in software to support data validation and exchange.


### Human Brain Project

 [humanbrainproject.eu/](https://www.humanbrainproject.eu/en/)
 
The Human Brain Project (HBP) is building a research infrastructure to help advance neuroscience, medicine, and computing. It is one of four FET (Future and Emerging Technology) Flagships, the largest scientific projects ever  funded by the European Union. The 10-year Project began in 2013 and directly employs some 500 scientists at more than 100 universities, teaching hospitals, and research centres across Europe.


### Human Immunology Project Consortium (HIPC)

[hipc.org](https://www.immuneprofiling.org/hipc)


The Human Immunology Project Consortium (HIPC) program was established in 2010 by the NIAID Division of Allergy, Immunology, and Transplantation as part of the overall NIAID focus on human immunology. Through this program, well-characterized human cohorts are studied using a variety of modern analytic tools, including multiplex transcriptional, cytokine, and proteomic assays; multiparameter phenotyping of leukocyte subsets; assessment of leukocyte functional status; and multiple computational methods. The HIPC program provides centralized research resources and comprehensive, centralized databases that utilize formal data standards and includes the Immunology Database and Analysis Portal (ImmPort) (https://www.immport.org/), offering advanced information technology support in the archiving and exchange of scientific data as an extensive data warehouse containing experimental data and metadata.


### Immune Epitope Database and Analysis Project

[iedb.org](https://iedb.org)

The Immune Epitope Database (IEDB) is a freely available resource funded by NIAID. It catalogs experimental data on antibody and T cell epitopes studied in humans, non-human primates, and other animal species in the context of infectious disease, allergy, autoimmunity and transplantation. The IEDB also hosts tools to assist in the prediction and analysis of epitopes.


### Informed Consent Ontology (ICO)

[](https://github.com/ICO-ontology/ICO)

Informed Consent Ontology (ICO) represents the domain of informed consent, including: consent forms, policies governing informed consent, agents working with patients and biospecimens accompanied by consent, and the process of informed consent itself. ICO aims to support informed consent data integration and reasoning in the clinical research space.


### Interlex

[Interlex.org](https://scicrunch.org/scicrunch/interlex/dashboard)

The InterLex project - a core component of SciCrunch and supported by projects such as the Neuroscience Information Framework project (NIF), the NIDDK Information Network (dkNET), and the Open Data Commons for Spinal Cord Injury - is a dynamic lexicon of biomedical terms. Unlike an encyclopedia, a lexicon provides the meaning of a term, and not all there is to know about it. InterLex is being constructed to help improve the way that biomedical scientists communicate about their data, so that information systems like NIF and dkNET can find data more easily and provide more powerful means of integrating that data across distributed resources.

### MicrobiomeDB: A Microbiome Resource 

[microbiomedb.org](www.microbiomedb.org)

MicrobiomeDB is a data-mining platform for interrogating microbiome experiments. MicrobiomeDB was developed as a discovery tool that empowers researchers to fully leverage their experimental metadata to construct queries that interrogate microbiome datasets. It is associated with VEuPathDB uses the OBI-based VEuPathDB ontology. Partial support for MicrobiomeDB has come from the Bill & Melinda Gates Foundation


### Neuroscience Information Framework

[neuinfo.org](http://neuinfo.org)

The Neuroscience Information Framework (NIF) is a dynamic inventory of Web-based neuroscience resources, data, and tools accessible via any computer connected to the Internet. An initiative of the NIH Blueprint for Neuroscience Research, NIF advances neuroscience research by enabling discovery and access to public research data and tools worldwide through an open source, networked environment.The NIF project is designed to serve the biomedical research community. The more participation from the community, the better the NIF. We welcome all feedback and suggestions and are actively looking for resource providers to make their resources accessible through the NIF. We offer a variety of methods for making resources searchable through the NIF. Collaborating institutions working on the current version of NIF include University of California, San Diego, California Institute of Technology, George Mason University, Yale University Medical College, and Washington University.


### SPARC.science

[sparc.science](https://sparc.science/)

The SPARC Portal is an international project funded by the NIH Common Fund’s SPARC program under OT3OD025349, OT3OD025348, and OT3OD025347. The SPARC Portal was created to help facilitate autonomic neuroscience research by hosting a growing collection of digital resources, including datasets, maps, and computational studies that focus on the role of the autonomic nervous system in controlling organ function. The resources available on the SPARC Portal are generated by SPARC program-supported research projects. Our long-term goal is to make this a resource the larger community can also contribute to. We are continually adding new datasets, maps, and computational studies to the site and are building new functionality to access and interact with those resources.


### Vaccination Informed Consent Ontology (VICO)

[VICO-ontology](https://github.com/VICO-ontology/VICO)

VICO is an application ontology for vaccination/immunization informed consent built upon previously constructed ICO (Informed Consent Ontology) and VO (vaccine ontology). The goal of developing VICO is to organize vaccination informed consent-related entities and their relations to enable vaccination informed consent data standardization and integration. The current focus of VICO delevelopment is to represent different vaccination informed consent forms in such a way that it will support consistent representation of immunization questionnaires.

### Vaccine Ontology

[vaccineontology](http://www.violinet.org/vaccineontology/)

The Vaccine Ontology (VO) is an ontology of vaccine and vaccination. VO utilizes the Basic Formal Ontology (BFO) and the Relation Ontology. Current focus is on vaccine categorization, vaccine components, vaccine quality, and vaccine-induced host responses. 
